docker/Dockerfile

```diff
@@ -1,6 +1,11 @@
 ARG NODE_VERSION
 ARG DOCKERHUB=dockerhub.tax.service.gov.uk
-FROM ${DOCKERHUB}/node:${NODE_VERSION:-10.15.1}-slim
+FROM ${DOCKERHUB}/node:${NODE_VERSION}-bullseye-slim
```

Comment:
can we document these workarounds somewhere?  why do we need to use this particular version?

Reply:
I'll put a comment, bullseye is the version name of the distro - I've dropped the default value because we get the version from .nvmrc and we don't want it to use any other value

Reply:
done